spec/models/review_spec.rb

```diff
@@ -18,7 +18,8 @@
 
   describe "validations" do
     it { should validate_numericality_of(:rating) }
-    it { should validate_inclusion_of(:rating).in_range(1..5) }
+    it { should validate_inclusion_of(:rating).in_range(1..5).
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.